### PR TITLE
Migrate centreon web to use centreon-ui's components (target: 19.10.x)

### PR DESCRIPTION
**Pull request by Thebarda**
# Pull Request Template

## Description
`@centreon/react-components` was renamed to `@centreon/ui` and centreon is still using `@centreon/react-components`. This PR integrate new UI components repository (`@centreon/ui`) instead of the old one (`@centreon/react-components`)

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [x] 19.10.x
- [ ] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Breadcrumb, Extension Manager and Navigation menu currently use centreon-ui components.
Through the navigation menu, go to Extension Manager then install an extension.
Use the breadcrumb by clicking on his own links.

#### Community contributors & Centreon team

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    '@babel/plugin-transform-arrow-functions',
+    '@babel/plugin-transform-destructuring',
+    '@babel/plugin-transform-function-name',
+    '@babel/plugin-transform-parameters',
+    '@babel/plugin-proposal-class-properties',
+    '@babel/plugin-transform-classes',
+    '@babel/plugin-transform-shorthand-properties',
+    '@babel/plugin-transform-regenerator',
+  ],
+  sourceType: 'unambiguous',
```
Comment (sc979) [inline babel.config.js]: What is this for ?

File: webpack.config.js [hunk 2]
```diff
@@ -113,42 +113,48 @@ module.exports = {
...
-              publicPath: '../../static/fonts/'
-            }
-        }]
+              publicPath: '../../static/fonts/',
+            },
+          },
+        ],
       },
       {
         test: /@centreon\/react\-components\/lib\/.+\.(bmp|png|jpg|jpeg|gif|svg)$/,
```
Comment (sc979) [inline webpack.config.js]: Shouldn't this path need to be modified too?

File: babel.config.js (hunk 1 above)
Comment (Thebarda) [reply babel.config.js]: `sourceType` consider a file as a module if it contains import and/or export

**Comment by bdauria** [comment]
LGTM, after https://jenkins.centreon.com/job/centreon-web/view/change-requests/job/PR-8210/ is successfully built.

**Comment by Thebarda** [comment]
@kduret Do you approve this PR ?

**Pull request merged**